src/EventStore.Client.Tests/Streams/EventDataComparer.cs

```diff
@@ -1,31 +1,34 @@
+using System;
```

Comment:
Need to create an issue to replace this with something that gives better error messages